res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -242,26 +242,22 @@ limitations under the License.
     }
```

Comment:
After this gets some updates from the review suggested, the issue description should get a table of **before** and **after** screenshots to easily compare the changes.

In this case, it's a bit more obvious that it's fixing a broken case [as pictured in the issue](https://github.com/vector-im/element-web/issues/22058) but it sounds like we're also going to fix the main timeline vertical positioning which is way more subtle.

Reply:
working on that